### DISCUSSION TITLE
Segítségkérés - D6CG3D

### DISCUSSION THREAD

**Discussion by pizoli** [SzFMV2021-Osz/warmup#11] [General]
Üdv @all!

Segítséget szeretnék kérni, mert a Pull request automatikus ellenőrzésénél hibát kapok!
A hibaüzenet:
```
Attempting to locate build script
  Error: Could not auto-detect a suitable build method
  Error: We were unable to automatically build your code. Please replace the call to the autobuild action with your custom build steps.  Failure invoking /opt/hostedtoolcache/CodeQL/0.0.0-20210909/x64/codeql/csharp/tools/autobuild.sh with arguments .
```
Milyen módosítás szükséges a helyes futáshoz?

Köszönettel
Z

@SzFMV2021-Osz/everyone @pintergreg

**Comment by pintergreg** [comment]
@pizoli Nem tudom mi volt a gond, egy rerun után végigfutott.
Egyébként nem a fordítást/tesztet végző job, hanem a statikus kódanalízisért felelős állt meg. Talán a szolgáltató oldalán volt valami gond. :shrug:

**Comment by pizoli** [comment]
@pintergreg 
Köszönöm!
Azt érzékeltem, hogy valami hatásomon kívüli probléma volt!
